application/backend/src/main/java/cmpe451/group12/beabee/login/controller/UserController.java

```diff
@@ -0,0 +1,24 @@
+package cmpe451.group12.beabee.login.controller;
```

Comment:
I didn't understand why did you move userController to login package. Isn't it supposed to be in user package?

Reply:
Since we are going to use feature based package structure, the package that most sense to put the user related classes is login package, however we can change however we want.